i18n/README.id.md

```diff
@@ -0,0 +1,91 @@
+<p align="center">
+<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/web/static/supabase-light-with-background.svg"/>
+</p>
+
+---
+
+# Supabase
+
+[Supabase](https://supabase.io) adalah sumber terbuka alternatif Firebase. Kita mengembangkan fitur Firebase menggunakan alat sumber terbuka kelas perusahaan.
+
+- [x] Menggunakan database Postgres yang di host
+- [x] Langganan waktu sebenarnya
+- [x] Otentikasi dan otorisasi
+- [x] API yang dibuat secara otomatis
+- [x] Dashboard
+- [x] Penyimpanan
+- [ ] Functions (segera hadir)
+
+## Dokumentasi
+
+Untuk dokumentasi lengkap, kunjungi [supabase.io/docs](https://supabase.io/docs)
+
+## Komunitas & Bantuan
+
+- [Forum Komunitas](https://github.com/supabase/supabase/discussions). Terbaik utnuk: membantu dengan mengembangkan, diskusi tentang praktik terbaik database.
+- [GitHub Issues](https://github.com/supabase/supabase/issues). Terbaik untuk: bug dan kesalahan yang kamu temui saat menggunakan Supabase.
+- [Email Support](https://supabase.io/docs/support#business-support). Terbaik untuk: masalah - masalah dengan database atau infrastrukturkamu
+
+## Status
+
+- [x] Alpha: Kita test Supabase dengan sejumlah pelanggan tertutup.
+- [x] Public Alpha: Siapa saja bisa daftar di [app.supabase.io](https://app.supabase.io). Tapi santai saja pada kami, ada beberapa masalah.
+- [x] Public Beta: Cukup stabil untuk sebagian besar kasus pengguna non-perusahaan
+- [ ] Public: Siap Produksi
+
+Kita sekarang sedang di Public Beta. Watch "rilis" dari repo ini untuk mendapatkan pembaruan besar.
+
+<kbd><img src="https://gitcdn.link/repo/supabase/supabase/master/web/static/watch-repo.gif" alt="Watch this repo"/></kbd>
+
+---
+
+## Bagaimana ini berkerja
+
+Supabase adalah kombinasi dari alat - alat sumber terbuka. Kita mengembangkan fitur Firebase menggunakan produk sumber terbuka kelas perusahaan. Jika alat - alat dan komunias - komunitas ada, dengan MIT, Apache 2, atau yag sama dengan open license, kita akan menggunakan dan dukung alat itu. Jika alat itu tidak ada, kita buat dan kita akan membuka kode itu. Supabase tidak 1 untuk 1 pemetaan Firebase. Tujuan kami adalah untuk memberi pengembang sebuah pengalaman seperti Firebase pengembang menggunakan alat -alat sumber terbuka.
+
+**Arsitektur saat ini**
+
+Supabase adalah sebuah [platform yang dihosting](https://app.supabase.io). Kamu bisa daftar dan mulai menggunakan Supabase tanpa menginstall apapun. Kita tetap membuat pengalaman mengembakan di lokal - ini sekarang tujuan utama kita, selama dengan platform yang stabil.
```

Comment:
- The standard translation for `install` is `pasang`. So you can replace `menginstall` with `memasang`.

- > selama dengan platform yang stabil.

  I don't think this is the correct translation.